### DISCUSSION TITLE
Remove user typed text within dropdown search functionality in Form component

### DISCUSSION THREAD

**Discussion by RipalPH** [sqlpage/SQLPage#704] [Q&A] [answered]
I have dropdown with options and searchable and multiple row level parameter set as true allowing multiple select. I need a way to remove typed text by user once the option is selected. How do you achieve that with in built component. For example, if I type Ba in the dropdown it displays Banana as option but upon selecting Banana, it needs to remove typed text ba. 

```sql
select 
    'form'                        as component,
    'examples/show_variables.sql' as action,
    'Reset'                       as reset;
select 
    'Fruits'         as label,
    'fruits[]'       as name,
    'select'         as type,
    TRUE             as multiple,
    TRUE             as create_new,
    'Good fruits...' as placeholder,
    TRUE             as searchable,
    'press ctrl to select multiple values' as description,
    '[{"label": "Orange", "value": 0, "selected": true}, {"label": "Apple", "value": 1}, {"label": "Banana", "value": 3, "selected": true}]' as options;
```

![image](https://github.com/user-attachments/assets/050c36a4-cd81-4969-a975-74c904048f15)

**Comment by lovasoa** [comment, marked as answer]
Oh, I'm realizing just now that I failed to follow-up here after I fixed the issue.

The problem is no longer present in the latest SQLPage releases :)

Sorry for the late reply.

**Reply by RipalPH** [reply]
No worries, I was able to pull new code and it worked. Thank you!
